### ISSUE TITLE
Missing references remain in test folder in v0.12.0-rc1

### ISSUE THREAD

**Issue by garborg** [julia-vscode/julia-vscode#792]
I had tried out 0.12-beta2 and ran into missing references throughout src and test.

I just upgraded to rc1 and all the missing references in src are fixed, but for example, any mention of `LanguageServer` or its contents in the test folder of 'LanguageServer.jl' is still a missing reference.

(Ran into the same thing in my own project, so I 'add'ed and 'dev'ed LanguageServer.jl in case there was a problem just with my project. Instantiated LanguageServer in case that made a difference, and verified the LanguageServer environment had been activated)

If there's any debug instructions I can follow that would be helpful, let m know.

**Comment by davidanthoff** [comment]
I think this is a problem only if you activate a package as the active project, right? If, on the other hand you have say a global env and dev the current package into that global env, and then make that global env the active env in VS Code, everything should work?

This is generally a bit tricky, because I think we really would need to handle packages that are activated as an env quite differently from other activated env... I don't think we'll be able to tackle that for v0.12.0 initially, and I'll have to think a bit how to best handle that in general.

**Comment by garborg** [comment]
That's correct -- thank you for the tip (I'll keep my projects active only in the repl for now)!

**Comment by davidanthoff** [comment]
I think the only scenario that doesn't work correctly right now is if you activate a package, activating non-package projects should work fine.
